csrankings.csv

```diff
@@ -14625,6 +14625,282 @@ Yvonne Jansen,University of Copenhagen,http://www.diku.dk/Ansatte/?pure=da%2Fper
+Gary Benson,Boston University,http://tandem.bu.edu/benson.html,E0GO0JkAAAAJ
+Christophe Giraud,Brigham Young University,http://dml.cs.byu.edu/,jHWj2yYAAAAJ
+Casey T. Deccio,Brigham Young University,https://casey.byu.edu/,NOSCHOLARPAGE
+Frank Jones,Brigham Young University,https://marriottschool.byu.edu/directory/details,NOSCHOLARPAGE
+Sherief Reda,Brown University,https://vivo.brown.edu/display/sreda,p69MQp4AAAAJ
+James Tompkin,Brown University,https://cs.brown.edu/courses/csci1430/,Ex3pgLAAAAAJ
+Andrew Stuart,California Institute of Technology,http://cms.caltech.edu/people/5854/profile,BQwkpB0AAAAJ
+Garth A. Gibson,Carnegie Mellon University,http://www.cs.cmu.edu/~garth/,WYctQAQAAAAJ
+Michael Rabinovich,Case Western Reserve University,http://engr.case.edu/rabinovich_michael/,30fyS8EAAAAJ
+Ming-Chun Huang,Case Western Reserve University,https://engineering.case.edu/about/school-directory/ming-chun-huang,NOSCHOLARPAGE
+Gregory S. Lee,Case Western Reserve University,http://engineering.case.edu/profiles/gsl7,NOSCHOLARPAGE
+Jim F. Martin,Clemson University,https://www.clemson.edu/madren/inn/,rUUtArEAAAAJ
+Xu Liu,College of William and Mary,http://www.cs.wm.edu/~xl10/,qhLT67EAAAAJ
+Bo Wu,Colorado School of Mines,http://inside.mines.edu/~bwu/,g6Wdt1YAAAAJ
+Adele E. Howe,Colorado State University,http://www.cs.colostate.edu/~howe/,FjV70BwAAAAJ
+Laura Moreno,Colorado State University,https://www.cs.colostate.edu/~lmorenoc/,UJM5NfoAAAAJ
+Louis-Noël Pouchet,Colorado State University,http://www.cs.colostate.edu/~pouchet/,TCppIZYAAAAJ
+Xi Chen,Columbia University,http://www.cs.columbia.edu/~xichen/Homepage/Welcome.html,y2pH4jcAAAAJ
+Angelos D. Keromytis,Columbia University,http://www.cs.columbia.edu/~angelos/,mncyWbcAAAAJ
+Jose Martinez,Cornell University,http://martinez.csl.cornell.edu/,NOSCHOLARPAGE
+Yudong Chen,Cornell University,https://people.orie.cornell.edu/yudong.chen/,ze5rCdwAAAAJ
+Amy Williams,Cornell University,http://williamslab.bscb.cornell.edu/,_-PAsvMAAAAJ
+Haiyuan Yu,Cornell University,http://yulab.icmb.cornell.edu/,NOSCHOLARPAGE
+Lei Xie,CUNY,http://compsci.hunter.cuny.edu/~leixie/,CujCxdYAAAAJ
+Alla Rozovskaya,CUNY,https://www.gc.cuny.edu/Page-Elements/Academics-Research-Centers-Initiatives/Doctoral-Programs/Computer-Science/Faculty-Bios/Alla-Rozovskaya,SL4bZHEAAAAJ
+Huy T. Vo,CUNY,https://serv.cusp.nyu.edu/~hvo/,HNwwWQgAAAAJ
+Qiang Liu,Dartmouth College,https://web.cs.dartmouth.edu/people/qiang-liu,XEx1fZkAAAAJ
+Bruce Donald,Duke University,http://www.cs.duke.edu/brd/,TkayRqQAAAAJ
+Kishor S. Trivedi,Duke University,https://trivedi.pratt.duke.edu/,fEVXhhAAAAAJ
+Jun Yang,Duke University,https://users.cs.duke.edu/~junyang/,NOSCHOLARPAGE
+Landon P. Cox,Duke University,https://users.cs.duke.edu/~lpcox/,zoBGvW4AAAAJ
+Benjamin Lee,Duke University,http://people.duke.edu/~bcl15/,4Tnj6PcAAAAJ
+Sina Farsiu,Duke University,http://people.duke.edu/~sf59/,mzcr92sAAAAJ
+Rong Ge,Duke University,https://users.cs.duke.edu/~rongge/,MVxcjEoAAAAJ
```

Comment:
Already in the database as:
* Rong Ge 0001,Duke University,https://users.cs.duke.edu/~rongge,MVxcjEoAAAAJ